awx/ui/test/e2e/cluster/docker-compose.yml

```diff
@@ -38,4 +38,5 @@ services:
     environment:
       AWX_E2E_CLUSTER_HOST: hub
       AWX_E2E_CLUSTER_PORT: 4444
+      E2E_RETRIES: ${E2E_RETRIES}
```

Comment:
I don't see a default value for this being set anywhere, what is the default?

Reply:
it would be set by the docker-compose invocation https://github.com/ansible/tower-qa/blob/6285e3c5dbd2af06434c69ce7cda8e610927e0d6/tools/jenkins/scripts/e2e.sh#L94

Reply:
removed implementation